plugins/decklink/decklink-output.cpp

```diff
@@ -66,6 +66,18 @@ static bool decklink_output_start(void *data)
 
 	DeckLinkDeviceMode *mode = device->FindOutputMode(decklink->modeID);
 
+	struct obs_video_info ovi;
+	if (!obs_get_video_info(&ovi)) {
+		LOG(LOG_ERROR,
+		    "Start failed - could not retreive obs_video_info!");
```

Comment:
I'm not fond of sentences using dashes.  Separate sentences or a colon are usually better options.  See elsewhere as well.